.vscode/extensions.json

```diff
@@ -1,6 +1,7 @@
 {
     "recommendations": [
         "eg2.tslint",
+        "ms-vscode.azure-account",
```

Comment:
I don't think azure account is ever used for ARM extension?

Reply:
Oh, I guess you're right.